src/main/java/net/sf/jabref/gui/SidePaneManager.java

```diff
@@ -104,6 +104,8 @@ public synchronized void hide(String name) {
             LOGGER.warn("Side pane component '" + name + "' unknown.");
         } else {
             hideComponent(sidePaneComponent);
+            frame.getCurrentBasePanel().getMainTable().setSelected(frame.getCurrentBasePanel().getMainTable().getSelectedRow());
+            frame.getCurrentBasePanel().getMainTable().requestFocus();
```

Comment:
Please extract `frame.getCurrentBasePanel().getMainTable()` to new local variable.